adoc/Private-Registry-Administration.adoc

```diff
@@ -743,3 +743,206 @@ suse-registry-harbor-registry-65854df7bc-mrfnj     2/2     Running   0
+
+When {spr} is configured to be deployed together with its internal services (database, Redis and filesystem storage for image and artifacts) Velero is able to fully cover the backup and restore for it. However, as Velero is only responsible for Kubernetes resources, when {spr} is deployed using external service providers, such as managed PostgreSQL database or object storage (Amazon S3 or Azure Blob Storage), additional actions must be performed to ensure the backup of data that is external to the Kubernetes cluster.
+
+In general, the backup and restore process for {spr} can be split in two parts:
+
+* Backup and restore of Kubernetes resources (performed by Velero):
+** Namespace
+** Deployment
+** ReplicaSet
+** StatefulSet
+** Endpoint
+** Service
+** Ingress
+** ConfigMap
+** Secret
+** PersistentVolumeClaim
+** PersistentVolume
+** Pod
+** Other CRDs (e.g. `Certificate` when using `cert-manager` or `RedisFailover` when using `redis-operator`)
+* Backup and restore of external services:
+** Database
+** Redis
+** Object Storage
+
+==== Backup and restore of Kubernetes resources
+
+The following steps describe how to perform the backup and restore of Kubernetes resources using Velero.
+
+===== Backup
+
+. Install Velero as described in the link:https://documentation.suse.com/suse-caasp/4.2/html/caasp-admin/_backup_and_restore_with_velero.html[documentation]
+
+. (optional) If you are using a volume provider that does not support volume snapshots or a volume-snapshot API, or does not have Velero-supported storage plugin, Velero leverages link:https://restic.net/[restic] as a generic solution to backup and restore this sort of persistent volume. To perform a Restic backup of persistent volumes, Velero requires the addition of a specific annotation (`backup.velero.io/backup-volumes=<VOLUME_NAME_1>,<VOLUME_NAME_2>,...`) to the pods that have the volumes mounted. This can be achieved by adding the following entries to the {spr} helm chart `harbor-values.yaml`, and then performing `helm upgrade` to apply the annotations:
++
+.harbor-values.yaml
+[source,yaml]
+----
+registry:
+  podAnnotations:
+    backup.velero.io/backup-volumes: registry-data
+trivy:
+  podAnnotations:
+    backup.velero.io/backup-volumes: data
+database:
+  podAnnotations:
+    backup.velero.io/backup-volumes: database-data
+redis:
+  podAnnotations:
+    backup.velero.io/backup-volumes: data
+----
++
+[source,bash]
+----
+helm -n registry upgrade suse-registry ./harbor -f harbor-values.yaml
+----
++
+.External Services
+[NOTE]
+====
+If using an external service, such as a managed PostgreSQL database, adding the `database` entry is not required. The same applies for `redis` when using an external Redis and for `registry` when using an external storage backend for storing images and artifacts.
```

Comment:
```suggestion
If using an external service, such as a managed PostgreSQL database, you do not need to add the `database` entry. The same applies for `redis` when using an external Redis, and for `registry` when using an external storage back-end for storing images and artifacts.
```

Reply:
We try to avoid passive voice where possible. One reason is that it is more difficult to translate.